charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -33,3 +33,17 @@ annotations:
       url: https://github.com/prometheus-community/helm-charts
     - name: Upstream Project
       url: https://github.com/prometheus-operator/kube-prometheus
+
+dependencies:
+- name: kube-state-metrics
+  version: "2.9.*"
+  repository: https://charts.helm.sh/stable
+  condition: kubeStateMetrics.enabled
+- name: prometheus-node-exporter
+  version: "1.12.*"
+  repository: https://prometheus-community.github.io/helm-charts
+  condition: nodeExporter.enabled
+- name: grafana
+  version: "5.8.*"
+  repository: https://grafana.github.io/helm-charts
+  condition: grafana.enabled
```

Comment:
I like it that you keep dependencies at the same version, while making the switch to helm 3.

Reply:
I plan to do a follow-up PR bumping grafana to a later version.

Reply:
Only thing is, updating to the newer version of Grafana is the only argument I can see for moving this chart to v2 now.